_posts/en/newsletters/2022-10-26-newsletter.md

```diff
@@ -0,0 +1,309 @@
+  (TabConf), several developers participated in a CoreDev.tech event.
+  [Transcripts][coredev xs] of several of the meetings from the event
+  have been provided by Bryan Bishop.  Notable discussions included:
+
+    - [Transport encryption][p2p encryption]: a conversation about the
+      recent update to the [version 2 encrypted transport
+      protocol][topic v2 p2p transport] proposal (see
+      [Newsletter #222][news222 bip324]).  This protocol would make it
+      harder for network eavesdroppers to learn which IP address
+      originated a transaction and improve the ability to detect and
+      resist man-in-the-middle attacks between honest nodes.
+
+        The discussion covers several of the protocol design
+        considerations and is a recommended read for anyone wondering
+        why the protocol authors made certain decisions.  It also
+        examines the relationship to the earlier [countersign][topic
+        countersign] authentication protocol.
+
+    - [Fee market][]: a wide-ranging discussion about transaction fees
+      historically, presently, and in the future.  Some topics included
+      queries about why blocks are seemingly almost always nearly full
+      but the mempool, debate about how long we have for a significant
+      fee market to develop before we have to [worry][topic fee sniping]
+      about Bitcoin's long-term stability, and what solutions we could
+      deploy if we did believe a problem existed.
+
+    - [FROST][]: a presentation about the FROST threshold signature
+      scheme.  The transcript documents several excellent technical
+      questions about the cryptographic choices in of the design and may
+      be interesting reading for anyone interested in learning more
+      about FROST specifically or cryptographic protocol design in
+      general.  See also the TabConf transcript about [ROAST][], another
+      threshold signature scheme for Bitcoin.
+
+    - [GitHub][github chat]: a discussion about moving the Bitcoin Core
+      project's git hosting from GitHub to another issue and PR
+      management solution, as well as considering the benefits of
+      possibly continuing to use GitHub.
+
+    - [Provable specifications in BIPs][hacspec chat]: a discussion
+      about using the [hacspec][] specification language in BIPs to
+      provide specifications which are provably correct.  See also the
+      [transcript][hacspec preso] for a related talk during the TabConf.
+
+    - [Package and v3 transaction relay][package relay chat]: the
+      transcript of a presentation about proposals to enable [package
+      transaction relay][topic package relay] and use new transaction
+      relay rules to eliminate [pinning attacks][topic transaction
+      pinning] in certain cases.
+
+    - [Stratum v2][stratum v2 chat]: a discussion that started with the
+      announcement of a new open source project implementing the Stratum
```

Comment:
```suggestion
      announcement of a new open-source project implementing the Stratum
```
Perhaps this should be added to the [compound words list](https://github.com/bitcoinops/bitcoinops.github.io/blob/master/STYLE.md#compound-words).